packages/angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -1,13 +1,32 @@
 import { Location } from '@angular/common';
-import { Directive, Attribute, Optional, SkipSelf, ElementRef, NgZone } from '@angular/core';
+import {
+  ViewChild,
+  ViewContainerRef,
+  Component,
+  Attribute,
+  Optional,
+  SkipSelf,
+  ElementRef,
+  NgZone,
+} from '@angular/core';
 import { Router, ActivatedRoute } from '@angular/router';
 import { IonRouterOutlet as IonRouterOutletBase } from '@ionic/angular/common';
 
-@Directive({
+@Component({
   selector: 'ion-router-outlet',
+  template: '<ng-container #outletContent><ng-content></ng-content></ng-container>',
 })
 // eslint-disable-next-line @angular-eslint/directive-class-suffix
 export class IonRouterOutlet extends IonRouterOutletBase {
+  /**
+   * static: true must be set so the query results are resolved
```

Comment:
```suggestion
   * `static: true` must be set so the query results are resolved
```
Easier to read in this suggestion.